### ISSUE TITLE
Vox blue wizard hat issue

### ISSUE THREAD

**Issue by Carthusia** [ParadiseSS13/Paradise#11858]
**Problem Description**:
What is the problem?

The wizard hat almost turns into a plague doctor mask that appears behind the vox's head

**What did you expect to happen**: 
Why do you think this is an issue?
Looks really off

**Why is this bad/What are the consequences:**
Why do you think this is an important issue?

Just another vox clothing issue to be noted

**Steps to reproduce the problem**:
The most important section. Review everything you did leading up to causing the issue.
Wear a blue wizardhat as a vox

**When did the problem start happening**:
If your report is about something that used to work but no longer does, when was the last time you remember it working?

Soon as i tried to wear a blue wizards hat

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was anything else out of the ordinary happening?)**:
Anything else you can tell us.

https://i.gyazo.com/19f77f4783a6d4715e776c95f92d615e.png

Simply put ona blue wizard hat as a vox and it will do this, but only for the side sprites
